contrib/database/sql/exec_trace_test.go

```diff
@@ -0,0 +1,155 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+// TODO: skip if Go >= 1.21, until gotraceui handles those format changes?
+
+package sql
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"net/http"
+	"runtime/trace"
+	"testing"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/contrib/database/sql/internal"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/httpmem"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	gotraceui "honnef.co/go/gotraceui/trace"
+)
+
+func TestExecutionTraceAnnotations(t *testing.T) {
+	if trace.IsEnabled() {
+		t.Skip("execution tracing is already enabled")
```

Comment:
hmm what could cause this test to be skipped here? Is this just a guard against other tests possibly leaving runtime tracing on?

Reply:
This is a guard against somebody running the tests with tracing already enabled, like `go tool test -trace=trace.out`.